e2e/tests/init_test.go

```diff
@@ -5,26 +5,37 @@ package e2e
 import (
 	"fmt"
 	"regexp"
+	"strconv"
 	"strings"
 	"testing"
+	"time"
 
 	"github.com/regen-network/gocuke"
 	"github.com/stretchr/testify/require"
 )
 
-var addrRe *regexp.Regexp
+var (
+	addrRe           *regexp.Regexp
+	amountRe         *regexp.Regexp
+	accNameToAddrMap = make(map[string]string)
+	keyRingFlag      = "--keyring-backend=test"
+)
 
 func init() {
-	addrRe = regexp.MustCompile(`address:\s+(pokt1\w+)`)
+	addrRe = regexp.MustCompile(`address: (\S+)\s+name: (\S+)`)
+	amountRe = regexp.MustCompile(`amount: "(.+?)"\s+denom: upokt`)
```

Comment:
It looks like previous version was expressing whitespace in terms of `\s+`,  my assumption would be to mitigate some shell formatting thing from causing a false negative match due to multiple spaces, a tab, or other whitespace character(s) instead of ` `. Wdyt about the following?

```suggestion
	addrRe = regexp.MustCompile(`address:\s+(\S+)\s+name:\s+(\S+)`)
	amountRe = regexp.MustCompile(`amount:\s+"(.+?)"\s+denom:\s+upokt`)
```